.github/workflows/snapshot-release.yml

```
@@ -33,14 +33,45 @@ jobs:
         if: github.event_name == 'issue_comment'
         id: comment-branch
 
+      - name: Get sha
+        id: parse-sha
```

Since this workflow can be dispatched by either `workflow_dispatch` or `issue_comment` events, we can skip this step if it wasn't triggered by a comment:

```suggestion
        if: github.event_name == 'issue_comment'
        id: parse-sha
```